test/cmd/newapp.sh

```diff
@@ -61,6 +61,7 @@ os::cmd::expect_success_and_text 'oc new-app ruby-helloworld-sample -o yaml' 'MY
 os::cmd::expect_success_and_text 'oc new-app ruby-helloworld-sample -o yaml' 'ADMIN_USERNAME'
 os::cmd::expect_success_and_text 'oc new-app ruby-helloworld-sample -o yaml' 'ADMIN_PASSWORD'
 os::cmd::expect_success_and_text 'oc new-app ruby-helloworld-sample --param MYSQL_PASSWORD=hello -o yaml' 'hello'
+os::cmd::expect_success_and_text 'oc new-app ruby-helloworld-sample --param MYSQL_PASSWORD@test/testdata/template-value-file.txt -o yaml' 'fileval'
```

Comment:
You might want to test failure scenarios. What happens if the file doesn't exist, can't be accessed given permission issues, file is empty, malformed filename like `MYSQL_PASSWORD@http://foo.com`, etc? Unit tests in `ParseEnvironmentArguments` for that works too.